frontend/packages/helm-plugin/src/components/list-page/AdminHelmRepositoriesListPage.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { Link, useParams } from 'react-router-dom-v5-compat';
+import NamespacedPage, {
+  NamespacedPageVariants,
+} from '@console/dev-console/src/components/NamespacedPage';
+import { PageHeading } from '@console/internal/components/utils';
+import RepositoriesPage from './RepositoriesListPage';
+
+type PageContentsProps = {
+  namespace: string;
+};
+
+const PageContents: React.FC<PageContentsProps> = ({ namespace }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <PageHeading title={t('helm-plugin~Helm Repositories')} className="co-m-nav-title--row">
+        <div>
+          <Link
+            className="co-m-primary-action"
+            to={`/helm-repositories/ns/${namespace || 'default'}/~new/form`}
```

Comment:
Do I have a chance to change the namespace from `default` to something else in the form?

Reply:
No, the namespace selector is disabled in the form.

Reply:
Let's open a follow up issue. `default` is rarely what you'd want, but I think it's good to get this change merged and address it later.